bin/cronjob

```diff
@@ -72,10 +72,12 @@ end
 #
 def main()
   command = ARGV[0]
-  result = `#{command} 2>&1`.strip # `2>&1` capture both STDOUT and STDERR into result
-  status = $?.exitstatus
-  hipchat_notify command, status, result, ARGV[1]
-  email_notify command, status, result, ARGV[1]
+  stdout, stderr, status = Open3.capture3(command)
+  result = stdout + stderr
+  exitstatus = status.exitstatus
+
+  honeybadger_notify command, exitstatus, stdout, stderr
+  email_notify command, exitstatus, result, ARGV[1]
```

Comment:
I was kind of wondering if using Open3 instead of `2>&1` might fix the problem with emails not containing any useful information. Probably not that relevant because if the useful information is in honeybadger that's better than email.